database/config/config.js

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
```

Comment:
I ca not see  .env file in your project

Reply:
we can not expose our .env file, instead, we can create .env.example file which will contain our environment variables only without their actual values.

e.g: 
POSTGRES_USER=
POSTGRES_DB=